### ISSUE TITLE
Failed to 'import numpy as np' in lpython

### ISSUE THREAD

**Issue by gtarcoder** [lcompilers/lpython#2814]
<img width="553" alt="image" src="https://github.com/user-attachments/assets/a536cdc5-107d-432a-9ed0-1140dac64997">

As the above image shows.

**Comment by kmr-srbh** [comment]
Yes, import aliasing is currently unsupported in LPython. We have an issue for it at #2026.

**Comment by patelalay231** [comment]
can I work on this?

**Comment by kmr-srbh** [comment]
@patelalay231 please go ahead.

**Comment by amritamishra01** [comment]
Hi! @kmr-srbh I'd like to work on this issue.

I noticed the last activity was over a year ago and no PR was opened, so I’d like to pick this up and work on a fix for import numpy as np alias support.

Please let me know if that’s okay — I can start immediately.
Thanks!
